check.go

```diff
@@ -287,4 +298,24 @@ func init() {
 		}
 		return nil
 	})
+
+	// Scan neulevel-1.0 extension
+	path = "epp > response > extension > " + ExtNeulevel10 + " extension > unspec"
+	scanResponse.MustHandleCharData(path, func(c *xx.Context) error {
+		dcd := &c.Value.(*response_).DomainCheckResponse
+		check := &dcd.Checks[len(dcd.Checks)-1]
+		charge := DomainCharge{Domain: check.Domain}
+		data := string(c.CharData)
+		pairs := strings.Split(data, " ")
+		for _, pair := range pairs {
+			parts := strings.SplitN(pair, "=", 2)
+			if parts[0] == "TierName" {
+				charge.Category = parts[1]
```

Comment:
Currently we don't have any data in the struct other than Domain, Category and CategoryName.